### DISCUSSION TITLE
Copy paste is not working in some apps

### DISCUSSION THREAD

**Discussion by abdulhajiyev** [end-4/dots-hyprland#521] [General]
For example I can copy from firefox but can't paste to discord, chrome, vs code, But If I copy from chrome I can paste to Discord and vs code.

**Comment by end-4** [comment]
hyprland issue, happens with xwayland apps

**Comment by ruru-m07** [comment]
I can copy from VSCode, and it shows in the clipboard as well. However, when I try to paste it somewhere, it only pastes an empty white space.

ctrl + c ( in vscode or discord )
mod + v
press enter
ctrl + v

:)

cc: @end-4 @abdulhajiyev -Is there a solution for this?

**Comment by justalike** [comment]
can confirm, still an issue.
same thing  but with code-oss

**Reply by ruru-m07** [reply]
Yeh @justalike for me this just works. 

```bash
code --password-store=gnome --enable-features=UseOzonePlatform --ozone-platform=wayland .
```

u can also edit your `/home/YOURUSER/.config/fish/config.fish` so u no need to write all args everytime. 

```fish
function code
    command code --password-store=gnome --enable-features=UseOzonePlatform --ozone-platform=wayland $argv
end
```

:)

**Reply by end-4** [reply]
doesn't launching in wayland mode make it (or any other chromium app) take forever to launch?

**Reply by ruru-m07** [reply]
Oh BTW, I don't use code OSS. I use VSCode (arch `x64 .tar.gz`) ![hmmm](https://cdn.discordapp.com/emojis/1295592576124194896.webp?size=48&name=hmmm)
